content/src/licenses/homemaker-home-health-aide.md

```diff
@@ -8,20 +8,26 @@ summaryDescriptionMd: If you are interested in continuing to employ individuals
   or facility care), you must renew your registration with the Employment and
   Personnel Services within the Division of Consumer Affairs online.
 ---
+
 If you are interested in continuing to employ, place, arrange for the placement of, or in any way refer, an individual to provide companion care services, health care services, or personal care services in the personal residence of a person with a disability or who is age 60 or older. You must renew your registration with the Division of Consumer Affairs online.
 
 ### Renewal Requirements
 
-* New `surety bond|surety-bond`/proof of bond continuation or certified financial report if the business has a net worth of $100,000 or more
-* Criminal record of any officers, directors, principals, or people with an ownership interest of 10 percent or more
-* General liability insurance certificate
-* NJ License of a Health Care Practitioner Supervisor, Registered Nurse, or Licensed Physician
-* Certificate of Accreditation
-* Applicable fees paid
-
-> **Once completed, you will have:**
->
-> A certificate of registration and registration number, beginning with “HP” and followed by 7 digits
->
-> **What if I don't renew my registration?**\
-> You may be required to pay penalties and/or ordered to cease and desist from operating as a Health Care Service Firm until you have obtained registration.
\ No newline at end of file
+- New `surety bond|surety-bond`/proof of bond continuation or certified financial report if the business has a net worth of $100,000 or more
+- Criminal record of any officers, directors, principals, or people with an ownership interest of 10 percent or more
+- General liability insurance certificate
+- NJ License of a Health Care Practitioner Supervisor, Registered Nurse, or Licensed Physician
+- Certificate of Accreditation
+- Applicable fees paid
+
+:::callout{ icon="false" calloutType="conditional" }
+
+- A certificate of registration and registration number, beginning with “HP” and followed by seven digits
+
+:::
+
+:::callout{ header="What if I don't renew my registration?" icon="false" calloutType="conditional" }
+
+You may be required to pay penalties and/or ordered to cease and desist from operating as a Health Care Service Firm until you have obtained registration.
+
```

Comment:
I wanna double check in the actual applications but its looks kinda like this may have been one block all together?

Reply:
I'm having trouble finding this one? It also looks like it might not actually be in our application based on the CMS notes.